packages/wonder-blocks-dropdown/components/dropdown-portal-mounter.js

```diff
@@ -0,0 +1,158 @@
+// @flow
```

Comment:
Hmm, yeah. I think you can save it for a follow-up PR but I think this is a good candidate for a new package (especially since it depends upon wonder-blocks-modal). Maybe something like `wonder-blocks-portal-mounter`?

Reply:
@jeresig are you worried too much stuff ending up in `wonder-blocks-core`?

Reply:
I don't know that we necessarily should share code just because it looks the same. There's a lot to be said for not coupling things if we don't need to, and I'd lean toward that here.

Reply:
I'm using React's portal instead, so this will be removed!